source/Octopus.Tentacle.Tests.Integration/Support/TentacleConfigurationsAttribute.cs

```diff
@@ -153,6 +135,22 @@ static class TentacleConfigurationTestCases
             return CombineTestCasesWithAdditionalParameters(testCases, additionalParameterTypes);
         }
 
+        static IEnumerable<Type> GetScriptServicesToTest(ScriptServiceVersionToTest scriptServiceToTest, Version? version)
+        {
+            return scriptServiceToTest switch
+            {
+                ScriptServiceVersionToTest.Version1 => new[] { ScriptServiceV1Type },
+                ScriptServiceVersionToTest.Version2 => new[] { ScriptServiceV2Type },
+                ScriptServiceVersionToTest.Version3Alpha => new[] { ScriptServiceV3AlphaType },
+                //if no specific script service version was specified, fallback on the services in the tentacle version
+                ScriptServiceVersionToTest.None => version != null
```

Comment:
Will this add test variations for Test that don't use script service? E.g. a test just interested in file transfer service will now have some doubled up tests

Reply:
E.g. 
![image](https://github.com/OctopusDeploy/OctopusTentacle/assets/86938706/497573e3-1747-46a0-96a0-7a5718e9367c)

Reply:
We may need an opt-in way to stop this from happening for tests that really don't want these variations

Reply:
hm yeah. Let me take a quick look

Reply:
@nathanwoctopusdeploy This has been fixed in [193b4df](https://github.com/OctopusDeploy/OctopusTentacle/pull/678/commits/193b4df0808c86d4cb807f259d30bd65bf747f58)

We can now opt out of per-script-service testing with the enum value `None`.